cmd/agent/app/reporter/tchannel/builder.go

```diff
@@ -18,7 +18,7 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-	"github.com/uber/tchannel-go"
+	tchannel "github.com/uber/tchannel-go"
```

Comment:
why this change?

Reply:
this was added automatically by my IDE somehow. I will revert it.